tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -81,6 +81,10 @@
       "5.*.*"
     ],
     "VersionConditions": [{
+      "MinVersion": "5.0.0",
+      "MaxVersionExclusive": "5.1.1",
+      "ExcludeTargetFrameworks": ["net461", "netcoreapp2.1", "netcoreapp3.0", "netcoreapp3.1", "net5.0", "net6.0"]
+    },{
```

Comment:
Added this because there's no compatible combination of packages that works with our sample because GraphQL.Server.All requires minimum of 5.1.1.